test/journeys/lib/events.js

```diff
@@ -29,3 +29,19 @@ export function getEventLog(myBrowser) {
   });
   return result.value;
 }
+
+/**
+ * Searches events object for matching event name
+ *
+ * @export
+ * @param {object} options
+ * @param {array} options.events
+ * @param {string} options.eventName
+ * @returns {(array|boolean)}
```

Comment:
This should always return an array, correct? When would it return boolean?

Reply:
derp, forgot to update the jsdocs after I made some changes.